app/models/HateoasUserAnswersSummary.scala

```diff
@@ -18,8 +18,14 @@ package models
 
 import play.api.libs.json.{JsObject, Json}
 
+import java.time.LocalDateTime
+import java.time.temporal.ChronoUnit.DAYS
+
 object HateoasUserAnswersSummary {
 
+  private def expiresInDays(ttlInDays: Int, createdAt: LocalDateTime): Long =
+    DAYS.between(LocalDateTime.now(), createdAt.plusDays(ttlInDays))
```

Comment:
Do we want to use a `clock` here to ensure they have **exactly** 30 days from the point of creation?

Reply:
Actually I see we're not using a clock at all! That's probably my bad

Reply:
Ah so we currently create the user answers in the frontend and then instantly submit them to the backend. Might be a bit of refactoring to move that logic to the backend. Happy to approve what we've got though 👍